contracts/EverlongStrategyKeeper.sol

```diff
@@ -169,12 +169,12 @@ contract EverlongStrategyKeeper is Ownable {
         }
 
         // Check if tend should be called.
-        (bool shouldTend, bytes memory calldataOrReason) = CommonReportTrigger(
+        (bool _shouldTend, bytes memory calldataOrReason) = CommonReportTrigger(
```

Comment:
Nit: Our style is only to have underscores in front of parameters, internal functions, and private storage values. Let’s rename this to `shouldTend_` and make this change elsewhere